### PR TITLE
Add home page

### PR DESCRIPTION
**Pull request by JPCatze**
This request add home page to the project

### CODE REVIEW

File: src/app/home/page.tsx
Comment (INZDY) [inline src/app/home/page.tsx:1]: Need to add redirect logic to this file too.
see "page protection" in
- (pages)/auth/page.tsx
or
- chat/page.tsx

**Review by INZDY — COMMENTED** [review]
Add redirect logic and default route.
Currently we have auth as home page.
Rearrage the route.

File: src/app/home/page.tsx
Comment (JPCatze) [reply src/app/home/page.tsx:1]: does the fa8a79f fix the problem?

Comment (INZDY) [reply src/app/home/page.tsx:1]: Not really.
Easiest fix would be letting the home be on "/" (default) route.
And for auth to have its own "/auth" route.
Finally,
`if (!data.session) {
    return redirect("/auth");
  }`



Simply:
- All current "/" -> "/auth"
then
- home page route -> "/"

Try doing those and test navigating around a bit.

But honestly, the homepage is absolutely insignificant. If you don't want to deal with it, just don't care.
We can (should) comeback later and do this last.

Comment (JPCatze) [reply src/app/home/page.tsx:1]: then I'll just come back to this again after we finish up all other frontend page

**Pull request closed without merging**